app/components/blacklight/search_context/server_item_pagination_component.rb

```diff
@@ -12,35 +12,50 @@ def initialize(search_context:, search_session:, current_document:)
       end
 
       def render?
-        @search_context.present? && (@search_context[:prev] || @search_context[:next]) && (@search_session['document_id'] == @current_document_id)
+        return false unless @search_context.present? && @search_session['document_id'] == @current_document_id
+
+        total == 1 || (@search_context[:prev] || @search_context[:next])
+      end
+
+      def display_separator
```

Comment:
Can we move the calls to this method out of the template and put them in `link_to_previous_document` and `link_to_next_document`? Then we don't need to have a conditional on `total == 1` in this method.

Reply:
On second thought, that may break compatibility for someone who is expecting this not to have a delimiter in it.  Disregard this.